lib/adal.js

```diff
@@ -532,8 +572,8 @@ var AuthenticationContext = (function () {
     };
 
     /**
-     * Logout user will redirect page to logout endpoint.
-     * After logout, it will redirect to post_logout page if provided.
+     * Redirects user to logout endpoint.
+     * After logout, it will redirect to post_logout page if specified.
```

Comment:
post_logout -> post_logout_redirect_uri (or whatever is the name of the property)